tests/performance_tests/performance_test_basic.py

```diff
@@ -161,13 +165,15 @@ def launchCluster(self):
             genesisPath=self.clusterConfig.genesisPath,
             maximumP2pPerHost=self.clusterConfig.maximumP2pPerHost,
             maximumClients=self.clusterConfig.maximumClients,
-            extraNodeosArgs=self.clusterConfig.extraNodeosArgs
+            extraNodeosArgs=self.clusterConfig.extraNodeosArgs,
+            prodsEnableTraceApi=self.clusterConfig.prodsEnableTraceApi,
+            specificExtraNodeosArgs=self.clusterConfig.specificExtraNodeosArgs
             )
 
     def setupWalletAndAccounts(self):
         self.wallet = self.walletMgr.create('default')
         self.cluster.populateWallet(2, self.wallet)
-        self.cluster.createAccounts(self.cluster.eosioAccount, stakedDeposit=0)
+        self.cluster.createAccounts(self.cluster.eosioAccount, stakedDeposit=0, validationNodeIndex=1)
```

Comment:
Hardcoding this to 1 going to cause problems if we ran with multiple producers

Reply:
It won't cause problems but it is incomplete if the intent is to enable it on all producer nodes.  The first node after the bios node will always have producers on it, even if it doesn't have the full complement of them.

Reply:
Clayton is correct, simply forgot to go back after testing and remove the hardcoded for the dynamic variable for our validation node.

Reply:
Addressed: [38f4076](https://github.com/AntelopeIO/leap/pull/385/commits/38f4076c8f698a5b6fb45f30643e061996b921f4)

This will become more dynamic as the test topology builds out to support more producers and validation nodes, etc.  For now just use the known producer and validation node IDs since the test is currently only set up to use one of each.

Reply:
And a fix: [bd95fbc](https://github.com/AntelopeIO/leap/pull/385/commits/bd95fbc845f4a209f6c06c03e9aca826c8ca22f6)

Reply:
I'm not sure I understand. I was able to run multi-producer tests with `performance_test.py` using `-p`. Does this break that? Seems like you could use `totalNodes - 1` for validation node index and just make sure `totalNodes` is always one more than configured producer nodes.

Reply:
While the underlying Regression Test Framework supports multi-producer configurations, and the Performance Harness is build on top of that infrastructure, the Performance Harness MVP does not fully support multi-producer tests yet as it was out of scope for the MVP.  It was built in such a way that the lift to get it to multi-producer is not significant, but the scope for the MVP was limited.  Thus, while it appears and accepts Regression Test Framework underlying arguments for multi-producer, etc., the Performance Harness still expects a single producer and single validation (non-producer) node.

Reply:
Realized minimal support for `-p` and `-n` would not be difficult, so implemented here: [68ee2ea](https://github.com/AntelopeIO/leap/pull/385/commits/68ee2eac5c3b4632acb0e82a30730dcc851ea39f)